package.json

```diff
@@ -115,7 +115,7 @@
     "docs": "jsdoc -c conf-api.json",
     "lint": "eslint --ext .js,.mjs extras scripts src test rollup.config.js",
     "serve": "serve build -l 51000",
-    "test": "mocha --recursive --require test/fixtures.mjs",
+    "test": "mocha --recursive --require test/fixtures.mjs \"test/asset/asset-list-loader.test.mjs\"",
```

Comment:
I'm guessing you need to back out this change?

Reply:
Yep, done 👍